src/jit/Backend.cpp

```diff
@@ -670,6 +674,12 @@ JITCompiler::JITCompiler(Module* module, int verboseLevel)
     if (sljit_has_cpu_feature(SLJIT_HAS_CMOV)) {
         m_options |= JITCompiler::kHasCondMov;
     }
+
+    if ((JITFlags & JITFlagValue::JITVerboseColor) == JITFlagValue::JITVerboseColor) {
```

Comment:
No need the `== JITFlagValue::JITVerboseColor)`. Non-zero result represent the bit is set. Btw, are these bits now?